.prettierrc

```diff
@@ -1,5 +0,0 @@
-{
-  "trailingComma": "all",
-  "singleQuote": true,
-  "plugins": ["@chanzuckerberg/prettier-config-edu"]
```

Comment:
prettier-config-edu isn't a plugin, so adding it here wasn't actually using it. I think 🤔 

It at least wasn't using it for the rules, but may have been for the Tailwind plugin?